datalad/support/gitrepo.py

```diff
@@ -3664,14 +3751,15 @@ def _save_add(self, files, git_opts=None):
                     # while there is no git-annex underneath here, we
                     # tend to fake its behavior, so we can also support
                     # this type of messaging
-                    message='\n'.join(r['error-messages'])
-                    if 'error-messages' in r else None,
+                    #message='\n'.join(r['error-messages'])
+                    #if 'error-messages' in r else None,
+                    message=None,
```

Comment:
note: indeed `r` cannot have `error-messages` in its return since it is 

```
    @staticmethod
    def _process_git_get_output(stdout, stderr=None):
        """Given both outputs (stderr is ignored atm) of git add - process it

        Primarily to centralize handling in both indirect annex and direct
        modes when ran through proxy
        """
        return [{u'file': f, u'success': True}
                for f in re.findall("'(.*)'[\n$]", ensure_unicode(stdout))]
```
so functionally this change would be correct and without side-effect.